third-party/libtm/libtm/src/Device.cpp

```diff
@@ -1797,6 +1797,11 @@ namespace perc {
         Bulk_Message msg((uint8_t*)&request, request.header.dwLength, (uint8_t*)&response, sizeof(response), mEndpointBulkMessages | TO_DEVICE, mEndpointBulkMessages | TO_HOST);
 
         mDispatcher->sendMessage(&mFsm, msg);
+        // ERROR_FW_INTERNAL just means we were unable to set the node
+        if (msg.Result == toUnderlying(Status::ERROR_FW_INTERNAL))
+        {
+            return Status::ERROR_FW_INTERNAL;
```

Comment:
Is this the only case that FW would return ERROR_FW_INTERNAL ?
If not - it would be hard to discriminate the root cause.

Reply:
@ev-mp Yes, for this message type